shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -37,23 +37,67 @@ public class ShadowBiometricManager {
   protected int canAuthenticate() {
     if (RuntimeEnvironment.getApiLevel() >= R) {
       return reflector(BiometricManagerReflector.class, realBiometricManager).canAuthenticate();
+    } else if (canAuthenticateInternal(0, BiometricManager.Authenticators.BIOMETRIC_WEAK)
+        != BiometricManager.BIOMETRIC_SUCCESS) {
+      return canAuthenticateInternal(0, BiometricManager.Authenticators.BIOMETRIC_WEAK);
```

Comment:
Why this method called twice with the same input parameters?

Reply:
I think i can update it better in this part 👌